scripts/article.ts

```diff
@@ -0,0 +1,64 @@
+import fs from "fs";
+import matter from "gray-matter";
+import path from "path";
+
+import { sync } from "glob";
+
+import dayjs from "scripts/dayjs";
+
+const articlesPath = path.join(process.cwd(), "articles");
+
+/**
+ * Fetches a list of all slugs (i.e. URL endpoints of articles) in the server.
+ * @returns a list of all slugs available
+ */
+export async function getSlugs() {
```

Comment:
This should have a return type attached to it.

Reply:
Same for all the functions here.

Reply:
It'll be some kind of `Promise<>` nonsense.

Reply:
I'm going to disagree with this - return type will be, as you said, ugly / nonsensical / even impractical - and it is far less important than declaring the types of the parameters.

Reference: https://www.totaltypescript.com/tips/dont-use-return-types-unless

Reply:
That makes sense to me. I accept that.